### ISSUE TITLE
Teleport

### ISSUE THREAD

**Issue by CrazyDra6on** [Jimk72/Icarus_Mods#7]
when i teleport i get a black screen and cant move 
![tele issue](https://user-images.githubusercontent.com/51029421/210469609-c9aa1400-58b1-45da-bb9b-502c172fd6c4.PNG)

**Comment by Jimk72** [comment]
We're you on dedicated server when it happened?

**Comment by CrazyDra6on** [comment]
yes on a dedicated server i don't know if it was the long distance or what but it happens once and a while when i go from one of my bases to another don't know if that's how you ment the tele to work but that's how i am using it. i just restart the game and it works for a bit then happens again then i just restart again.

**Comment by posicat** [comment]
This happened to me playing solo on my home machine, I have also ended up under the ground, before the new textures/resolutions loaded in ... One base this happened to regularly, I would need to walk as far as I could in the building to trigger the game reloading it's polygons/textures screen, and then everything was fine.  My new base doesn't have this problem.
